### PR TITLE
Update installation of HNPWA-vanilla documentation

### PR DESCRIPTION
**Pull request by kresnasatya**
Hi Cristian, I just try update installation of HNPWA-vanilla documentation. You can try in local. If it runs well, please let me know. Thanks a lot.

### CODE REVIEW

**Comment by cristianbote** [comment]
Hey @satyakresna,

Thank you! 😄 This clears up the confusing parts of running it locally, for sure. I need to check something first, because I think you can do it via `firebase-cli` and don't need to have the `python` installed.

I'll get back to you.
Thanks again!

**Comment by kresnasatya** [comment]
You're welcome Cristian. Nice to help the author and me how it works.

**Comment by cristianbote** [comment]
Hey @satyakresna, I just did a commit and added the `npm run serve` command. Would you be willing to update the wording on it? I'm gonna add some comments.

File: README.md [hunk 1]
```diff
@@ -13,10 +13,14 @@ Made with ❤️ in Cluj-Napoca, Transylvania 🏰
 
 1. Clone the repo    
 1. Run `npm install`
-1. Serve it locally with `npm run serve`
-    * Running _production_ locally `npm run start-prod`
+1. Run
+    * `npm run copy-assets` 
+    * `npm run html-minify`
+    * `npm run bundle`
+    * `npm run build-sw`
```
Comment (cristianbote) [inline README.md]: Instead of these lines, we could run the `npm run build-prod` and end up in the same place.

File: README.md [hunk 2]
```diff
@@ -13,10 +13,14 @@ Made with ❤️ in Cluj-Napoca, Transylvania 🏰
...
+1. It will generate a `public` folder with assets (html, css, js) inside them. Move to `public` folder and run `python -m SimpleHTTPServer 8080`.
```
Comment (cristianbote) [inline README.md]: This is no longer needed, you can run `npm run serve` and would spun up a firebase locally served server.

**Review by cristianbote — CHANGES_REQUESTED** [review]
Looking forward you to your reply.

**Comment by kresnasatya** [comment]
Hi @cristianbote , just finished based on your request. Done, feel free to check. Thanks a lot.

**Review by cristianbote — APPROVED** [review]

**Comment by cristianbote** [comment]
Awesome @satyakresna!

**Pull request merged**